.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -40,34 +40,22 @@ body:
 - type: textarea
   id: Description
   attributes:
-    label:  What did you do? Describe the bug
+    label:  Describe the bug and the steps to reproduce it
     description: |
       A clear and concise description of the bug.
-
+      - What actually happened? Please include the actual, raw output. You are encouraged to submit a PR that reproduces this in `packages/cli/test/functional/test_site/bugs/index.md`.
```

Comment:
> You are encouraged to submit a PR that reproduces this in `packages/cli/test/functional/test_site/bugs/index.md`

Should we consider removing this? I don't recall anyone doing so recently and the steps below ask for code for reproduction, though I can see how the PR might be easier for investigation

Reply:
I agree with your observation =>
- a PR reproduction might aid in the investigation
- but people don't seem to be doing it now, most probably because it's too troublesome

Do you think it will be good to improve this aspect by writing up a note in our DG (section: how to create bug report)?
- will be a separate issue

Reply:
I'm think we shouldn't have need a "how to create bug reports" section in the DG since the purpose of the issue templates is to teach/enforce creating bug reports in a constructive way, and we should minimise the friction of creating a bug report (i.e. shouldn't need to read DG). 

other considerations:
- most of the time since code snippets are usually sufficient for a minimal reproduction. 
- too many test PRs could clog up the PRs tab
- complex issues might need a PR to reproduce though - is there any value to having the reproduction PR in our Markbind repo vs inspecting code on the bug reporter's repo directly?

Reply:
Also keep in mind that bug reporters may be end users who are not necessarily programmers.

In general, use a philosophy that the most important thing is to get the user to initiate a communication with us (i.e., post _something_) because our assumption is that every such communication benefits us (i.e., it can potentially help us improve the product) more than the person posting the issue. A partial bug report is better than no bug report. If we can figure out the missing information ourselves, we do that. But if we can't, we can always respond with a request for more information.

Reply:
I see where you're coming from.

I feel there's value in reproduction PR because I do see that being suggested in some of the big OSS projects out there, but obviously, this takes effort on the bug reporter's end.

> most of the time since code snippets are usually sufficient for a minimal reproduction.
> complex issues might need a PR to reproduce though - is there any value to having the reproduction PR in our Markbind repo vs inspecting code on the bug reporter's repo directly?

Indeed only complex cases require something more than code snippets. In some cases, bug reporters may not have a public repo.

> too many test PRs could clog up the PRs tab

This might be ok as devs can filter and see the latest list of PRs by date (similar to what we can do for issues).

My understanding from this convo so far is that it's not necessary most of the time and we should probably not impose upon devs to do this reproduction.

As currently the instruction is stated as an "encouragement" and it's not prominent, do you think it's fine to keep it that way for advanced devs who need it? What do you think?

Reply:
Update: I have adjusted the wording to make it clear that it's not necessary to reproduce the bug via PR.
![image](https://user-images.githubusercontent.com/41845017/215692972-bc040e42-8703-481d-ac32-40e257a7844f.png)

Reply:
Looks good! Thanks @tlylt :)